netbox/utilities/constants.py

```diff
@@ -27,3 +27,17 @@
     ('111111', 'Black'),
     ('ffffff', 'White'),
 )
+
+# Keys for PostgreSQL advisory locks. These are arbitrary bigints used by
+# the pg_advisory_lock contextmanager/decorator. When a lock is acquired,
+# one of these keys will be used to identify said lock.
+#
+# When adding a new key, pick something arbitrary and unique so
+# that it is easily searchable in query logs.
+ADVISORY_LOCK_KEYS = {
+    'available-prefixes': 123456710,
```

Comment:
Is there any reason that these specific integers were chosen? I realize that they are arbitrary, but would it make sense to use somewhat more consecutive values (similar to what we've done for other constants)?

Reply:
I changed them to some simpler, but also easy to search for numbers, let me know what you think